### ISSUE TITLE
Does pitmp-maven-plugin support JUNIT5?

### ISSUE THREAD

**Issue by rdelgatte** [STAMP-project/pitmp-maven-plugin#13]
#### Characteristics
- **Issue Type**: [test report]
- **Reproducibility**: [always]
- **Severity**: [feature]
- **Tool/Service/Component**: [PIT 1.4.0]
- **Execution Environment**: [Maven 3.5.2 - Java 8 - JUNIT5]
- **Reporter**: [Rémi Delgatte - remi.delgatte@gmail.com]

#### Description
I'd like to test `pitmp-maven-plugin` but it looks like it does not work well with *JUNIT5* (I had the same behavior when setting up PIT before adding the plugin for junit5).
Here is the stacktrace I get: 

```
18:06:41 PIT >> INFO : MINION : 18:06:41 PIT >> INFO : Checking environment

18:06:41 PIT >> INFO : MINION : 18:06:41 PIT >> SEVERE : JUnit was not found on the classpath. PIT requires JUnit 4.6 or above.
See http://pitest.org for more details.

18:06:41 PIT >> SEVERE : Error generating coverage. Please check that your classpath contains JUnit 4.6 or above.
```
Does `pitmp-maven-plugin` supports JUNIT5?

Thank you 

Rémi

**Comment by vmassol** [comment]
I don't know about PITMP and JUnit5 but for PIT and JUnit5, see https://github.com/pitest/pitest-junit5-plugin

**Comment by rdelgatte** [comment]
Yes I know, it is already set up and works really well but I would have like to benefit from PITMP because my project is composed of several modules.

**Comment by vmassol** [comment]
Yep I agree. I have the same needs too on XWiki. We're currently using PIT with JUnit5 but if we want to try out PITMP we'll also need support for JUnit5.

**Comment by vmassol** [comment]
It's really a pity that PITMP is not developed as a feature of the PIT Maven plugin.... That's so sad IMO.

**Comment by Cael35** [comment]
I try to follow PIT releases, but I haven't tried JUnit5 yet. I'll try to take care of it quickly! :-)

**Comment by Cael35** [comment]
I just released the v1.3.2 which should fix this issue.
